docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -647,19 +661,19 @@ For this tutorial we ignore these warnings. Click on the following link for a tu
 The tictactoe directory now contains two new files, `tictactoe.wasm` and `tictactoe.abi`.
 
 ### Deploy the Single Node Testnet
-[Deploy the smart contract](https://developers.eos.io/manuals/eos/v2.0/cleos/how-to-guides/how-to-deploy-a-smart-contract) on the single node testnet with the following commandline.
+[Deploy the smart contract](https://developers.eos.io/manuals/eos/v2.0/cleos/how-to-guides/how-to-deploy-a-smart-contract) on the single node testnet tictactoe account with the following command.
 
 In the same directory as the generated `wasm` and `ABI` files run
 ```shell
-cleos set contract host ./ tictactoe.wasm tictactoe.abi -p host@active
+cleos set contract tictactoe ./ tictactoe.wasm tictactoe.abi -p tictactoe@active
 ```
 
 ## Play The Game
 Now that the smart contract has been successfully deployed [push smart contract actions](https://developers.eos.io/manuals/eos/v2.0/cleos/command-reference/push/push-action) to the blockchain to play the game.  
 
 
 ### Create a Game
-A game requires a host and a challenger. Use the accounts created earlier in the “Create the necessary accounts and key pairs” section of the tutorial for these. These accounts use arbitrary names. In this example assume the host has the account name of `host` and the challenger has the account name of `challenger`.
+A game requires a host and a challenger. Use the accounts created earlier in the “Create the necessary accounts and key pairs” section of the tutorial for these. These accounts can use arbitrary names. In this example assume the host has the account name of `host` and the challenger has the account name of `challenger`.
```

Comment:
“Create the necessary accounts and key pairs” it would be nice to have this as a link to that section header